### DISCUSSION TITLE
网站登录加载不出来问题

### DISCUSSION THREAD

**Discussion by wzfzyjcj** [halo-dev/halo#5564] [General]
### 你当前使用的版本

2.13.1

### 描述一下此特性

就是在登https://github.com/halo-dev/halo后回到https://github.com/halo-dev地方后就会经常出现登陆不了，当使用VPN后就可以加载出来了，那么可以解决这个问题，不使用VPN还可以加载出来。其次去到别的页面后回到前一个页面加载不出来也解决掉

**Comment by ruibaby** [comment]
不太明白你的意思，这是 GitHub，与此项目没有关系。

/triage unresolved
